packages/blockchain-wallet-v4-frontend/src/data/components/dex/sagas.ts

```diff
@@ -295,21 +325,23 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
   }
 
   const fetchSwapQuoteOnChange = function* (action) {
-    const { field, form } = action?.meta
-
-    // exit whenever the counterTokenAmount changes, to avoid infinitely calling fetchSwapQuote
-    if (field === 'counterTokenAmount' || field === 'step') return
-
-    // reset error if user changes token
-    if (field === 'baseToken') {
-      const error = yield select(selectors.components.dex.getSwapQuote)
-      if (error) yield put(A.clearCurrentSwapQuote())
-    }
+    const { field, form, touch } = action?.meta
```

Comment:
Careful here - this will not work if `meta` is `undefined`. It will throw something like "can't destructure from undefined"

Reply:
good call out